packages/visualizations/src/components/Table/constants.ts

```diff
@@ -7,3 +7,24 @@ export const DATA_FORMAT = {
     boolean: 'boolean',
     url: 'url',
 } as const;
+
+/* potential other rtl languages: ['iw', 'kd', 'pk'] */
+export const RTL_LANGUAGES_CODE = [
+    'ar',
+    'az',
+    'fa',
+    'he',
+    'jv',
+    'kk',
+    'ks',
+    'ku',
+    'ml',
+    'ms',
+    'ps',
+    'sd',
+    'so',
+    'tk',
+    'ug',
+    'ur',
+    'yi',
+] as string[];
```

Comment:
as const probably (it's already string[]). I didn't find a good way to fetch this list from somewhere reliable. At the very list we want a source link as comment.